modular_skyrat/modules/modular_items/lewd_items/code/lewd_organs/_genital.dm

```diff
@@ -11,21 +9,7 @@
 	/// The currently inserted sex toy.
 	var/obj/item/inserted_item
 
-/// Helper proc for checking if internal fluids are full or not.
-/obj/item/organ/genital/proc/internal_fluid_full()
-	return internal_fluid_count >= internal_fluid_maximum
-
-/// Adds the given amount to the internal fluid count, clamping it between 0 and internal_fluid_maximum.
-/obj/item/organ/genital/proc/adjust_internal_fluid(amount)
-	internal_fluid_count = clamp(internal_fluid_count + amount, 0, internal_fluid_maximum)
-
-/// Tries to add the specified amount to the target reagent container, or removes it if none are available. Keeps in mind internal_fluid_count.
-/obj/item/organ/genital/proc/transfer_internal_fluid(datum/reagents/reagent_container = null, attempt_amount)
-	if(!internal_fluid_datum || !internal_fluid_count || !internal_fluid_maximum)
-		return FALSE
-
-	attempt_amount = clamp(attempt_amount, 0, internal_fluid_count)
-	if(reagent_container)
-		reagent_container.add_reagent(internal_fluid_datum, attempt_amount)
-	internal_fluid_count -= attempt_amount
-
+/obj/item/organ/genital/Initialize(mapload)
+	. = ..()
+	create_reagents(internal_fluid_maximum, REAGENT_HOLDER_ALIVE)
+	reagents.maximum_volume = internal_fluid_maximum
```

Comment:
```suggestion
```
The first arg of `create_reagents` sets `maximum_volume` of the resulting container so this is redundant